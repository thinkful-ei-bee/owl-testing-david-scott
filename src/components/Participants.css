
.participantCard img {
  border-radius: 50%;
  display: inline-block;
  width: 30px;
  height: 30px;
}

.sideBar {
  float: left;
  min-height: 600px;
  width: 300px;
  overflow: scroll;
}

.indented {
  color: black;
  text-indent: 25px;
}

.dot {
  height: 10px;
  width: 10px;
  background-color: rgb(26, 224, 26);
  border-radius: 50%;
  display: inline-block;
  margin-right: 10px;
}

.greydot {
  background-color: grey;
}