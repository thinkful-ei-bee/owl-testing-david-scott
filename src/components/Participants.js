import React from 'react';
import './Participants.css';

class Participants extends React.Component {
  render() {
    return <div />;
  }
}

export default Participants;