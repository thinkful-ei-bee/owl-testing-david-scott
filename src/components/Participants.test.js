import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Participants from './Participants';

const participants = [
  {
      id: 1,
      name: 'Koren Templeton',
      avatar:
          'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1',
      inSession: true,
      onStage: true
  },
  {
      id: 2,
      name: 'Caty Flucker',
      avatar:
          'https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1',
      inSession: true,
      onStage: false
  },
  {
      id: 3,
      name: 'Axe Kubicka',
      avatar: 'https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1',
      inSession: false,
      onStage: true
  }];


describe('<Participants />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Participants participants={participants}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });  

  it('matches previous snapshot', () => {
    const tree = renderer.create(<Participants participants={participants} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});
