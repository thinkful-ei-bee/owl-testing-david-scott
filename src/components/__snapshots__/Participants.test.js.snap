// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participants /> matches previous snapshot 1`] = `<div />`;
